package.json

```diff
@@ -29,7 +29,8 @@
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
   "peerDependencies": {
     "bookshelf": ">=0.8.2",
-    "hapi": "11.x"
+    "hapi": "13.x",
```

Comment:
Since the original bar of `11.x` was because of the lack of `hapi-qs`, what do you think of making this `>=12.x`?